docs/component-guides/KServe.md

```diff
@@ -0,0 +1,181 @@
+
+For this tutorial, create a certificate by following [this ACM document](https://docs.aws.amazon.com/acm/latest/userguide/gs-acm-request-public.html#request-public-console) for `*.platform.example.com` and `*.staging.platform.example.com`(both domains in the same certificate) in the region where your cluster exists. Certificate is only valid upon [validation of domain ownership](https://docs.aws.amazon.com/acm/latest/userguide/domain-ownership-validation.html). After successful validation, the certificate status will be `Issued`. Export the ARN of the certificate created:
+1. `export certArn=<>` 
+
+If you use Dex as auth provider in your Kubeflow deployment, refer the [Dex specific section](#dex---update-the-load-balancer). If you use Cognito, continue to the next section
+
+## Cognito - Create a HTTP header evaluation based Load Balancer
+
+Currently, it is not possible to programatically authenticate a request through ALB that is using Amazon Cognito to authenticate users. (i.e. you cannot generate the `AWSELBAuthSessionCookie` cookies by using the access tokens from Cognito). Instead of using the existing ALB, we will be creating another Application Load Balancer(ALB) endpoint which authorizes based on custom strings specified in a predefined HTTP header. We will use an ingress to set [HTTP header conditions](https://docs.aws.amazon.com/elasticloadbalancing/latest/application/load-balancer-listeners.html#http-header-conditions) on ALB which will configure rules that route requests based on the HTTP headers in the request. This can be used for service to service communication in your application.
+
+### Create Ingress
+
+1. Configure the following parameters for [ingress](../../awsconfigs/common/istio-ingress/overlays/api/params.env):
+    - certArn: ARN of certificate created in previous step
+    - (optional) httpHeaderName: Custom HTTP header name you want to configure for the rule evaluation. Default is `x-api-key`
+    - httpHeaderValues: one or more match strings that need to be compared against the header value in the request received. You only need to pass one of the tokens in the request. Pick strong values.
+    1. Replace the `token1` string with a token of your choice and optionally the httpHeaderName
```

Comment:
so basically ALB will make sure it only serves the request with `token1`( due to header  `x-api-key` ) entry.

Reply:
yes